Sources/SwiftIDEUtils/Syntax+Classifications.swift

```diff
@@ -41,30 +40,31 @@ public extension SyntaxProtocol {
   /// intersect the provided `range`.
   ///
   /// - Parameters:
-  ///   - in: The relative byte range to pull ``SyntaxClassifiedRange``s from.
+  ///   - in: The range to pull ``SyntaxClassifiedRange``s from.
   /// - Returns: Sequence of ``SyntaxClassifiedRange``s.
-  func classifications(in range: ByteSourceRange) -> SyntaxClassifications {
+  func classifications(in range: Range<AbsolutePosition>) -> SyntaxClassifications {
     return SyntaxClassifications(_syntaxNode, in: range)
   }
 
-  /// The ``SyntaxClassifiedRange`` for a relative byte offset.
+  /// The ``SyntaxClassifiedRange`` for an byte offset.
   /// - Parameters:
-  ///   - at: The relative to the node byte offset.
-  /// - Returns: The ``SyntaxClassifiedRange`` for the offset or nil if the source text
+  ///   - at: The byte offset measured relative to the syntax tree's root.
+  /// - Returns: The ``SyntaxClassifiedRange`` for the offset or `nil`` if the source text
   ///   at the given offset is unclassified.
+  @available(*, deprecated, message: "Use classification(at: AbsolutePosition) instead")
```

Comment:
This deprecation feels a little odd since it actually has different behavior. Should we make that clear in the deprecation message?

Reply:
Good point. I changed it to maintain the same behavior it had before the PR and kept the deprecation message the same.